### PR TITLE
Add Governance document

### PR DESCRIPTION
**Pull request by gabro**

### CODE REVIEW

File: packages/website/docs/Governance/governance.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Bento is used by design and development teams both at buildo and outside of buildo.
+
+These guidelines apply to **both**: regardless of whether you work at buildo or not, this is the recommended way of interacting with the Bento project and team.
+
+:::
+
+## 🐛 Bug reports
+
+Bugs with the React library or the Figma toolkit must be reported by [**opening an issue**](https://github.com/buildo/bento-design-system/issues/new) on the GitHub repository (be sure to follow use the appropriate bug report template).
```
[the remark this answers is not in the document]
Comment (gabro) [reply packages/website/docs/Governance/governance.md]: Fixed, thanks

File: packages/website/docs/Governance/governance.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## 🙏 Support requests
+
+If you have a doubt on how to use Bento to create a design (whether you are a developer implemeting a design, or a designer using the Figma toolkit), please use the **Q&A category** on [GitHub discussions](https://github.com/buildo/bento-design-system/discussions/new).
+
+## 🤝 External contributions
+
+By default, issues are worked on by the internal team at buildo.
+
+If you are interested in contributing a pull request, we recommend discussing your intention to do so in an issue, so that we can advise on how to proceed, and maximize the changes that your pull request gets merged.
```
[the remark this answers is not in the document]
Comment (gabro) [reply packages/website/docs/Governance/governance.md]: Indeed...

**Review by federico-ercoles — COMMENTED** [review]
A couple of minor details in the comments, everything else looks ok.
Maybe it could be useful to also specify the language in which to submit issues and requests? Like use english universally, to avoid having discussions about features/bugs in italian where an external user may not contribute?

**Comment by federico-ercoles** [comment]
> > Maybe it could be useful to also specify the language in which to submit issues and requests? Like use english universally, to avoid having discussions about features/bugs in italian where an external user may not be able to contribute?
> 
> I kind of assumed English to the be default, since documentation, existing issues and so on are in English, and it's a public repo. Do you think it should be explicit?

I suggested that mainly to prevent internal feedback to be submitted in Italian (since from what I've seen we use English and Italian interchangeably in private repositories), but I agree it can easily be inferred.

**Pull request merged**